[2024-10-12T18:12:42.000+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-12T18:12:42.014+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first-dag.task1 scheduled__2021-08-10T00:00:00+00:00 [queued]>
[2024-10-12T18:12:42.022+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first-dag.task1 scheduled__2021-08-10T00:00:00+00:00 [queued]>
[2024-10-12T18:12:42.022+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-12T18:12:42.042+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2021-08-10 00:00:00+00:00
[2024-10-12T18:12:42.048+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first-dag', 'task1', 'scheduled__2021-08-10T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpvlfn924v']
[2024-10-12T18:12:42.050+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask task1
[2024-10-12T18:12:42.053+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=324) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-12T18:12:42.057+0000] {standard_task_runner.py:72} INFO - Started process 333 to run task
[2024-10-12T18:12:42.085+0000] {task_command.py:467} INFO - Running <TaskInstance: first-dag.task1 scheduled__2021-08-10T00:00:00+00:00 [running]> on host 50fe842129eb
[2024-10-12T18:12:42.147+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='first-dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2021-08-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-10T00:00:00+00:00'
[2024-10-12T18:12:42.148+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-12T18:12:42.163+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-12T18:12:42.167+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Running task1"']
[2024-10-12T18:12:42.173+0000] {subprocess.py:86} INFO - Output:
[2024-10-12T18:12:42.175+0000] {subprocess.py:93} INFO - Running task1
[2024-10-12T18:12:42.175+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-12T18:12:42.195+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-12T18:12:42.196+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first-dag, task_id=task1, run_id=scheduled__2021-08-10T00:00:00+00:00, execution_date=20210810T000000, start_date=20241012T181242, end_date=20241012T181242
[2024-10-12T18:12:42.237+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-12T18:12:42.246+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-12T18:12:42.248+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
